assignment-2/7s.md

```diff
@@ -1,2 +1,91 @@
+Their culture is based also on the people, there for their focus on motivation, community and trust rather than structure and control.
+Product development is based on Lean. They release their new features to a small group of customers limiting the 'blast radius' of a possible fail and it allows them to conduce a lot of small experiments. Of course it produces a lot of waste but this can be seen from a Lean perspective.
+They have something called 'core metrics data' and it is feed with a lot of dynamism, due their constant experimentation.
+
+##### Staffing
+
+Spotify has a very clear definition of what and how employees careers should develop in order to achieve the most important organization goals.[6] "Everyone in *Spotify tech* should have a way to grow their careers and expand the impact of their -work no matter what role they play".
+The Spotify technology career framework, consist on letting the employee transfer itself on as many roles and domain expertise as possible in order to gain experience and later evaluate their impact into the organization.
+*On recognition and promotions* Spotify observes constantly the impact someone has at a wider scope, and they have a formal ways to promote their employees
+*On setting the step for a new employee* Spotify gives 6 months to all new employees to get involved with their role and the organization.
+*On compensation* A formal promotion should lead to higher pay in a straightforward and transparent way.
+
+##### Skills
+
+Spotify seems to be good at keeping their organization dynamic, focusing on their community rather to an hierarchical structure.
+
+##### Style
+
+They have an Agile engineering culture and the autonomy that every member team has over their work, have evolve to a high rate percentage of team members satisfaction and they keep investing on it. Autonomy seems also to keep high motivation levels, teams are fast, and their minimize de dependency from and to, other teams.
+ 
+##### Shared Values
+
+Adding more functional and visionary features to their software product.
+Keeping the organization as a very nice place to work in.
+
+
+* **2.2 Non-Software industry:**
+* 
+### KLM
+Note: Most of the information about the leadership comes from talks with my (Ger) father, who is a copilot for KLM. Therefore there are no official sources to cite and the leadership style is largely from a pilot's point of view. So different departments will likely have other styles of internal management.
```

Comment:
I think we should reconsider whether we should use KLM as an organization for this report. From what I understand here there is no reference information. Thus, it is not possible to properly analyze this. I know it is short day, but I think it is best if we give a description of another organization. For example, a company which we may find more about is Heineken (example paper: http://www.emeraldinsight.com/doi/pdfplus/10.1108/09534810210429336). If we continue with KLM, we need to say a lot more about it than what we have now.